index.bs

```diff
@@ -209,119 +259,75 @@ interface Connect {
 };
 </pre>
 
-The `connect` method performs the following steps:
+The {{connect()}} method performs the following steps:
 
 <ol>
-  <li>A connection is established to the specified `SocketAddress` asynchronously.</li>
+  <li>A connection is established to the specified {{SocketAddress}} asynchronously.</li>
   <li>New {{Socket}} instance is created with each of its attributes initialised immediately.</li>
   <li>The created {{Socket}} instance is returned immediately.</li>
-  <li>If the connection fails for any reason, the socket's `closed` promise is rejected with a [[#socket-error]] exception describing why the connection failed.</li>
+  <li>If the connection fails for any reason, the socket's {{closed}} promise is rejected with a [=SocketError=] describing why the connection failed.</li>
   <li>The instance's {{ReadableStream}} and {{WritableStream}} streams can be used immediately.</li>
 </ol>
 
-At any point during the creation of the {{Socket}} instance, `connect` may throw a [[#socket-error]] exception. One
-case where this can happen is if the input address is incorrectly formatted.
-Errors which occur asynchronously- after the socket instance has been returned- must reject the
-socket's `closed` promise.
+At any point during the creation of the {{Socket}} instance, `connect` may throw a [=SocketError=]. One case where this can happen is if the input address is incorrectly formatted. Errors which occur asynchronously (after the socket instance has been returned) must reject the socket's {{closed}} promise.
```

Comment:
Nitpicking but it would be nice to keep to a 80 or 100 line length limit. (there are cases above as well where the wrapping is missing too)

Reply:
https://github.com/speced/bikeshed/issues/662#issuecomment-1332775833 :( so this wont exist in Bikeshed ever. A quick search also came up empty. My preference in this circumstance is to not wrap because then we'll have to manually check line lengths every time we make changes (and I guarantee we'll miss something). If you feel strongly I'm happy to manually wrap though 👍 doesn't matter that much to me